game-samples/generators/spanner/README.md

```diff
@@ -0,0 +1,17 @@
+# Spanner load generation
+
+This is a WIP to generate load directly to Spanner, instead of having a REST api.
```

Comment:
agreed. I may actually remove this file because it was a test at creating a custom Locust User to avoid having to write a REST service. Feedback from the sample-apps team indicated this is probably not necessary in the context of apps.